mote/projects/base/library/browser/patterns/atoms/text/data.yaml

```diff
@@ -1 +1,2 @@
+tag: span
```

Comment:
Made `text` similar to `nested` - this returns existing functionality by wrapping text in a span by default and preserving backwards compatibility. We can now use it to create more variety and offload more different types of text to it.